src/components/contest/ContestPage.tsx

```diff
@@ -257,30 +257,30 @@ const ContestPage = () => {
                 ariaLabel="three-dots-loading"
                 visible={true}
               />
-            ) : */}
-            {state.problemList.error.length > 0 ? (
-              <Alert key={"danger"} variant={"danger"}>
-                {state.problemList.error} Most probably because CF API is down.
-                Try reloading after few minutes. API link:
-                https://codeforces.com/api/problemset.problems .
-              </Alert>
-            ) : (
-              <ContestList
-                contestlist={
-                  randomContest === -1
-                    ? paginate()
-                    : [contestList.contests[randomContest]]
-                }
-                category={filter.category}
-                submissions={submissions}
-                showColor={filter.showColor}
-                showDate={filter.showDate}
-                showRating={filter.showRating}
-                perPage={filter.perPage}
-                pageSelected={selected}
-                theme={state.appState.theme}
-              />
-            )}
+            ) : <>
+              {state.problemList.error.length > 0 ? (
+                <Alert key={"danger"} variant={"danger"}>
+                  {state.problemList.error} Most probably because CF API is down.
+                  Try reloading after few minutes. API link:
+                  https://codeforces.com/api/problemset.problems .
+                </Alert>
+              ) : (
+                <ContestList
+                  contestlist={
+                    randomContest === -1
+                      ? paginate()
+                      : [contestList.contests[randomContest]]
+                  }
+                  category={filter.category}
+                  submissions={submissions}
+                  showColor={filter.showColor}
+                  showDate={filter.showDate}
+                  showRating={filter.showRating}
+                  perPage={filter.perPage}
+                  pageSelected={selected}
+                  theme={state.appState.theme}
+                />
+              )}</>}
```

Comment:
```suggestion
            ) : (
              <>
                {state.problemList.error.length > 0 ? (
                  <Alert key={"danger"} variant={"danger"}>
                    {state.problemList.error} Most probably because CF API is
                    down. Try reloading after few minutes. API link:
                    https://codeforces.com/api/problemset.problems .
                  </Alert>
                ) : (
                  <ContestList
                    contestlist={
                      randomContest === -1
                        ? paginate()
                        : [contestList.contests[randomContest]]
                    }
                    category={filter.category}
                    submissions={submissions}
                    showColor={filter.showColor}
                    showDate={filter.showDate}
                    showRating={filter.showRating}
                    perPage={filter.perPage}
                    pageSelected={selected}
                    theme={state.appState.theme}
                  />
                )}
              </>
            )}
```
Slight modification. Everything else on this page looks good. Nice work.

Reply:
updated the codee